### ISSUE TITLE
Log message improvment, if the agent name is already used

### ISSUE THREAD

**Issue by cheesi** [microsoft/azure-pipelines-agent#3708]
## Agent Version and Platform
2.196.2

OS: Windows Server 2019

## Azure DevOps Type and Version
dev.azure.com

Organization name shouldn't matter

## What's not working?
First of all this is not a bug, it's more an improvement suggestion.

Today we wanted to install the agent on a new server.
After calling the `config.cmd`, and following the steps, the installer asks you to enter the name of the agent.
If you enter a name, which is already in use for that agent pool, you will get a message something like "enter replacement" which doesn't really say, that the name is already in use.

In our case: apparently someone else already did the installation, so bad communication on our side. However, a better error message would still provide a better usability, as it was absolutely not clear, what the issue was and I only discovered this by accident, when I check the agent pool.

**Comment by anatolybolshakov** [comment]
Hi @cheesi could you please share more details about your request? Would you like agent name (specified during configuration) to be printed in the error message - when there is a name conflict?

**Comment by cheesi** [comment]
Hey @anatolybolshakov !

I can hardly remember.

The thing was, I tried to install the agent on a server, where it was already installed (with that exact name, as I was using the server name). And the message was just like "Enter replacement". It was no where clear that, 1) the name was already used and 2) that the agent was already installed on the server.

And it would have been good, to get that information. It's not like that I'm installing a new agent every other day, so I don't know the exact installation process, even the "Enter replacement" was already hard to spot and I only found it, because I checked the log from the last installation (on another server).

TL;DR: Usability is not very good, because those mentioned cases don't provide proper logging.

**Comment by anatolybolshakov** [comment]
@cheesi got it, thanks! Let us take a look - we can probably just update the message here for more relevant.

**Comment by ismayilov-ismayil** [comment]
@cheesi PR was merged. We'll update the thread when the fix is deployed
